frontend/src/components/MonacoEditor.tsx

```diff
@@ -24,9 +25,33 @@ interface DialogError {
   message: string
 }
 
+const GHConnected = ({ isGithubConnected }: { isGithubConnected: boolean }) => {
+  const githubConnected = () => {
+    return (
+      <Tooltip title="gitHub is connected. Saving will push to github">
+        <GitHub />
+      </Tooltip>
+    )
+  }
+
+  return (
+    <span
+      style={{
+        marginLeft: '1rem',
+        verticalAlign: 'middle',
+      }}
+    >
+      {isGithubConnected ? githubConnected() : 'Github is not connected'}
```

Comment:
```suggestion
      <Tooltip title="GitHub is connected. Saving will push to GitHub">
        <GitHub />
      </Tooltip>
    )
  }

  return (
    <span
      style={{
        marginLeft: '1rem',
        verticalAlign: 'middle',
      }}
    >
      {isGithubConnected ? githubConnected() : 'GitHub is not connected'}
```
Kinda nitpicky, but the way GitHub is written is very inconsistent. Doesn't really matter in code, but for UI it's better to keep things consistent.

Reply:
Nicely spotted. Which way would you prefer github to be written in?

Reply:
I noticed that you already did the changes so I could just accept it